docs/source/certificates/certificates.md

```diff
@@ -0,0 +1,173 @@
+
+<b>peer and orderer:</b> Signed certificates used by Hyperledger Fabric nodes (peers and orderers) and are node enrollment certificates. These certificates are either located physically on the node or mapped so they are visible to the node. Hyperledger Fabric nodes must have a valid certificate with the required attributes in order for the node to start.
+
+<b>client:</b> Signed certificates that allow signed requests passed to the Hyperledger Fabric nodes. These can be requests that do not require admin level certificates or client applications that need to pass signed requests to Hyperledger Fabric.
+
+#### TLS Certificates
+
+TLS certificates are signed certificates that allow Hyperledger Fabric nodes and clients to sign and encrypt communications. A valid TLS certificate is required for all communications.
+
+### Certificate Expiration
+
+All certificates (public keys) associated with enrollments are given an expiration date by the issuing certificate authority (CA). These dates must be monitored and enrollments must be renewed (by re-enrollment) before the expiration date of the certificate.
+
+### Prerequisites and resources
+
+The following Fabric documentation on identities, MSPs and CAs provide context for understanding this guide:
+
+Identity in Hyperledger Fabric:<br>
+https://hyperledger-fabric.readthedocs.io/en/latest/identity/identity.html#identity
+
+MSPs in Hyperledger Fabric:<br>
+https://hyperledger-fabric.readthedocs.io/en/latest/membership/membership.html#membership-service-provider-msp
+
+Registration and Enrollment:<br>
+https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/use_CA.html#overview-of-registration-and-enrollment
+
+Registering an Identity:<br>
+https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/use_CA.html#register-an-identity
+
+Enrolling an Identity:<br>
+https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/use_CA.html#enroll-an-identity
+
+
+It is key to understand that an identity contains the public certificate and private key used for encryption.  The public certificate is a signed certificate from the CA and is referred to generically as a certificate in the following table.
+
+The table shows the types of certificates used by Hyperledger Fabric.
+
+
+## Technical details
+
+**Note:** Certificates for Organization CAs and the TLS CA are being listed separately.  This is due to the differing purposes for the CAs.   The Organization CA supplies signing certificates and the TLS CA supplies TLS certificates for securing connections.
+
+### Organization CA Certificates
+
+| Certificate | Description | Location | Impact if expired | Reference |
+|----------------|-----|---------------------------|-----------|---|
+| Root Certificate | Controlling certificate used to sign all certificates issued by the CA. | Stored on disk in the CA directory.<br><br>ca-cert.pem | CA root certificates are issued for 15 years. A New certificate must be issued.  |  |
```

Comment:
Certificates are not used to sign. On the contrary, they are used to verify signatures on all certificates that the CA has issued (in case of a CA cert) or to verify signatures on all artefacts generated by the entity (e.g. transactions). Also, "controlling" doesn't have a lot of meaning in this context.

I would say that  the root certificate is publicly known and permits verification of all certificates issued by that CA.

Root CA certs are not only stored on disk but also in the channel configuration.